lms/djangoapps/grades/tests/test_models.py

```diff
@@ -490,3 +509,37 @@ def _assert_tracker_emitted_event(self, tracker_mock, grade):
                 'grading_policy_hash': str(grade.grading_policy_hash),
             }
         )
+
+    def test_clear_grade(self):
```

Comment:
What is this setup doing, specifically? I can see the test is to check that one is deleted and one isn't, but it's really hard to tell what the difference is between params and another_params.

Reply:
It looks like it's just the user? If that's the case I would suggest doing it like this, just to make that apparent to a reader:
```python
other_user = UserFactory.create()
other_user_params = {
  **self.params,
  'user_id': other_user.id
}
```

Reply:
And hey, how about you add another course grade, for params['user_id'] but a different course, and assert that that is also untouched